core/domain/email_manager.py

```diff
@@ -77,6 +77,46 @@ def log_new_error(*args, **kwargs):
         'subject': _PLACEHOLDER_SUBJECT,
         'html_body': _PLACEHOLDER_HTML_BODY,
     })
+
+_EDITOR_ROLE_EMAIL_SUBJECT = (
+    '%s added you to one of their explorations on Oppia.org')
+
+_EDITOR_ROLE_EMAIL_BODY = (
+    '%s added you to his exploration, \"%s\", as a %s.<br>'
+    '<br>This means you can:<br>'
+    '<ul>%s</ul><br>'
+    '<br>You can find exploration here: '
+    '<a href="http://www.oppia.org/%s">%s</a><br>'
+    '<br>Thank You for contributing to oppia<br>'
+    '<br>Best Wishes<br>'
+    'The Oppia Team<br>'
+    )
```

Comment:
I would suggest either deindenting this by 4 (to match the opening parenthesis) or move it to the end of the previous line; ditto below.